.circleci/config.yml

```diff
@@ -0,0 +1,308 @@
+            #  echo "WARNING: latest git tag could not be found"
+            #  echo "Please, make sure you fetch all tags from upstream with"
+            #  echo "the command ``git fetch --tags --verbose`` and push"
+            #  echo "them to your fork with ``git push origin --tags``"
+            #fi
+            # Build docker image
+            e=1 && for i in {1..5}; do
+              docker build \
+                --cache-from=neurostuff/nimare \
+                --rm=false \
+                -t neurostuff/nimare:latest \
+                --build-arg BUILD_DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"` \
+                --build-arg VCS_REF=`git rev-parse --short HEAD` \
+                --build-arg VERSION="${CIRCLE_TAG:-$THISVERSION}" . \
+              && e=0 && break || sleep 15
+            done && [ "$e" -eq "0" ]
+      - run:
+          name: Docker save
+          no_output_timeout: 40m
+          command: |
+            mkdir -p /tmp/cache
+            docker save neurostuff/nimare:latest \
+            | pigz -8 -p 3 > /tmp/cache/docker.tar.gz
+      - persist_to_workspace:
+          root: /tmp
+          paths:
+            - cache/docker.tar.gz
+            - src/NiMARE
+
+  update_cache:
+    machine:
+      # Ubuntu 14.04 with Docker 17.10.0-ce
+      image: circleci/classic:201711-01
+    working_directory: /tmp/src/NiMARE
+    steps:
+      - attach_workspace:
+          at: /tmp
+      - save_cache:
+         key: docker-v1-{{ .Branch }}-{{ epoch }}
+         paths:
+            - /tmp/cache/docker.tar.gz
+
+  test_python:
+    machine:
+      image: circleci/classic:201711-01
+    working_directory: /tmp/src/NiMARE
+    steps:
+      - checkout:
+          path: /home/circleci/src/NiMARE
+      - run:
+          name: Check whether build should be skipped
+          command: |
+            cd /home/circleci/src/NiMARE
+            if [[ "$( git log --format=oneline -n 1 $CIRCLE_SHA1 | grep -i -E '\[skip[ _]?test[s]?\]' )" != "" ]]; then
+              echo "Skipping python tests"
+              circleci step halt
+            fi
+      - attach_workspace:
+          at: /tmp
+      - run:
+          name: Check PyPi preconditions
+          command: |
+            pyenv global 3.5.2
+            virtualenv venv
+            pip install "setuptools>=27.0" cython numpy twine docutils
+            python setup.py check -r -s
+            python setup.py sdist
+
+      - run:
+          name: Load Docker image layer cache
+          no_output_timeout: 30m
+          command: |
+            docker info
+            set +o pipefail
+            if [ -f /tmp/cache/docker.tar.gz ]; then
+              sudo apt update && sudo apt -y install pigz
+              pigz -d --stdout /tmp/cache/docker.tar.gz | docker load
+              docker images
+            fi
+      - run:
+          name: Run pytest
+          command: |
+            docker run --rm neurostuff/nimare pytest -vs /src/NiMARE/ 
+
+  get_data:
```

Comment:
Do not need the get_data job for now, we will just download the data with pytest. so this can be removed